src/checks/checker.cc

```diff
@@ -811,7 +791,7 @@ void checker::run_sync(
   hst->set_latency(0.0);
 
   // Adjust check attempts.
-  adjust_host_check_attempt_3x(hst, true);
+  hst->adjust_check_attempt_3x(true);
```

Comment:
This method looks like the service one ::adjust_check_attempt(), maybe we could harmonize their names.